src/entities/repos/submission_repo.py

```diff
@@ -80,6 +82,15 @@ async def get_filing_tasks(session: AsyncSession) -> List[FilingTaskDAO]:
     return await query_helper(session, FilingTaskDAO)
 
 
+async def get_contact_info(session: AsyncSession, lei: str = None, filing_period: str = None) -> ContactInfoDAO:
+    filing_id = None
+    if lei and filing_period:
+        filing = await get_filing(session, lei=lei, filing_period=filing_period)
+        filing_id = filing.id
+    result = await query_helper(session, ContactInfoDAO, filing=filing_id)
```

Comment:
with the relationship built correctly in the dao layer, you won't need to do a separate query; it should just be part of the filing object